pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/fraser/barnepensjon/BarnepensjonForhaandsvarselFraser.kt

```diff
@@ -15,57 +14,70 @@ class BarnepensjonForhaandsvarselFraser {
                 text(
                     Language.Bokmal to "Som følge av vedtaket som er gjort har du fått <beløp> kroner for " +
                             "mye utbetalt i barnepensjon fra og med <dato> til og med <dato>.",
-                    Language.Nynorsk to "",
-                    Language.English to "",
+                    Language.Nynorsk to "Som følgje av vedtaket som er gjort, har du fått utbetalt <beløp> " +
+                            "kroner for mykje i barnepensjon frå og med <dato> til og med <dato>.",
+                    Language.English to "As a result of a decision of <dato>. You have received NOK <beløp> " +
```

Comment:
Her kjens det veldig mykje meir naturleg med komma enn punktum (reknar med at det er andre som står bak tekstane, altså)

Reply:
Stusset litt på den der selv. Tenkte det bare var jeg som var sær, men da er vi to. Hører med Mari.